src/components/PanelHeader/PanelHeader.tsx

```diff
@@ -1,19 +1,32 @@
 import { ReactNode } from "react";
-import { useQueryParam, StringParam } from "use-query-params";
+import { useQueryParams, StringParam } from "use-query-params";
 
 type Props = {
   title: ReactNode;
 };
 
+type QueryString = {
+  [key: string]: string | undefined;
+};
+
 export default function PanelHeader({ title }: Props): JSX.Element {
-  const [panelQs, setPanelQs] = useQueryParam("panel", StringParam);
+  const [panelQs, setPanelQs] = useQueryParams({
```

Comment:
Some new tests need to be added for this component to handle the fetching and resetting of the query params.